docs/Access Control with JWT Token.md

```diff
@@ -0,0 +1,40 @@
+# Access Control with JWT Token
+
+## Overview
+
+Clients using the api endpoint for tunnel `/v3/devices/{id}/services/{address}/connection` can use a jwt token to restrict access to a cloud application though the edge-proxy.
```

Comment:
Because the original API token is also acceptable, I think we should explicitly say that we are supporting two different flows.

Reply:
Yes, but this document is only for jwt token.

Reply:
Is this the only doc that we are giving out?

Reply:
I think we should delete this doc once the official document of the tunnel with is out, then put that documentation in the readme. This does not serve as a pelion-direct-connect-client doc.

Reply:
sounds good. Then never mind. Current info should be good enough